### PR TITLE
Add podcast page, powered by anchor RSS

### LINKED ISSUE #70: Add a home/feature for the podcast on apisyouwonthate.com
Where should the podcast live on the website? 
Should we have a blog post entry for each episode?

### PR DESCRIPTION
**Pull request by mbifulco**
Fixes #70 

This'll be easier once we have an episode published to anchor - I've used my own podcast's feed for the time being.  How's this look?

image

# what it is
- adds a new link in header to `/podcast`
- sources podcast episode metadata from anchor.fm via a gatsby plugin
- each episode is displayed in reverse-chronological order on `/podcast`
- there is currently no page for an individual episode (we should use Anchor for that, I think?)

# todo:
- need to update the rss URL in `gatsby-config.js` to point to our podcast's feed

### CODE REVIEW

**Comment by philsturgeon** [comment]
@mbifulco this looks great! let's fire this over to @matthewtrask for when he's got the podcast edited and uploaded. Hit 1000mi then do the thing! :)

File: gatsby-config.js [hunk 1]
```diff
@@ -17,6 +17,13 @@ module.exports = {
         path: `${__dirname}/src/images`,
       },
     },
+    // source podcast episodes from anchor.fm
+    {
+      resolve: 'gatsby-source-anchor',
+      options: {
+        rss: 'https://anchor.fm/s/2a4c840/podcast/rss',
```
Comment (philsturgeon) [inline gatsby-config.js]: @matthewtrask can you let me know what our podcast RSS is once you've uploaded it? I'll set up a redirect from apisyouwonthate.com/podcast/rss so we dont get screwed over like with apibusters.com

**Comment by philsturgeon** [comment]
@mbifulco we'll need the #79 before merging this too btw, sorry!

**Comment by mbifulco** [comment]
@philsturgeon I think you may need to get login credentials to @matthewtrask 👌

**Comment by philsturgeon** [comment]
Here’s our podcast URL! https://apisyouwonthate.com/podcast/rss

**Comment by philsturgeon** [comment]
merged!

**Pull request closed without merging**